index.ts

```diff
@@ -147,6 +147,12 @@ app.delete('/guests/:id', function (req, res) {
   res.json(guest);
 });
 
+// Clear entire list of guests
+app.delete('/guests', function (req, res) {
+  guestList.splice(0, guestList.length);
+  res.status(200);
```

Comment:
I would return something here, maybe the list of the deleted users?
or a message telling the deletion was complete